src/sass/App.sass

```diff
@@ -1,33 +1,37 @@
 .App
-  text-align: center
+  background-image: url(../assets/prayer.jpg)
```

Comment:
I'm not sure what effect we're going with, but I don't think this is what we want. If I'm not mistaken, this way the background of the web page will always be a static image. When we scroll the page, the image should move up as well. This means its not a background but an image on the page _(in a div or sth)_.